THANKS.txt

```diff
@@ -171,3 +171,4 @@ Agilent which gave a genereous donation for support of SciPy.
 UC Berkeley for providing travel money and hosting numerous sprints.
 The University of Stellenbosch for funding the development of
     the SciKits portal.
+Google Inc.
```

Comment:
This is IMO better but still feels a bit too inclusive: many people worked on scipy documentation without Google's support. 
In multi-author research papers it's reasonably widespread to phrase it along the lines of "TP acknowledges a partial support from...". Maybe you can model the acknowledgement along those lines?

At any rate, I'm ready to drop my objections if there are strong opinions.